### ISSUE TITLE
OracleEnhancedAdapter structure dump schema migrations multi insert is supported should dump schema migrations using multi inserts failure

### ISSUE THREAD

**Issue by yahonda** [rsim/oracle-enhanced#1893]
### Steps to reproduce
Refer https://travis-ci.org/rsim/oracle-enhanced/jobs/545973855

### Expected behavior
It should pass.

### Actual behavior

```ruby
Failures:
  1) OracleEnhancedAdapter structure dump schema migrations multi insert is supported should dump schema migrations using multi inserts
     Failure/Error:
       expect(dump).to eq <<~SQL
         INSERT ALL
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160101000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160102000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160103000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160104000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160105000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160106000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160107000000')
         INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160108000000')
     
       expected: "INSERT ALL\nINTO \"SCHEMA_MIGRATIONS\" (version) VALUES ('20160101000000')\nINTO \"SCHEMA_MIGRATIONS...60109000000')\nINTO \"SCHEMA_MIGRATIONS\" (version) VALUES ('20160110000000')\nSELECT * FROM DUAL\n"
            got: "INSERT ALL\nINTO \"SCHEMA_MIGRATIONS\" (version) VALUES (20160101000000)\nINTO \"SCHEMA_MIGRATIONS\"...20160109000000)\nINTO \"SCHEMA_MIGRATIONS\" (version) VALUES (20160110000000)\nSELECT * FROM DUAL\n"
     
       (compared using ==)
     
       Diff:
       @@ -1,13 +1,13 @@
        INSERT ALL
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160101000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160102000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160103000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160104000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160105000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160106000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160107000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160108000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160109000000')
       -INTO "SCHEMA_MIGRATIONS" (version) VALUES ('20160110000000')
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160101000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160102000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160103000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160104000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160105000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160106000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160107000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160108000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160109000000)
       +INTO "SCHEMA_MIGRATIONS" (version) VALUES (20160110000000)
        SELECT * FROM DUAL
```

### System configuration
**Rails version**:master branch

**Oracle enhanced adapter version**:master branch

**Ruby version**:2.6.3

**Oracle Database version**:11.2

**Comment by yahonda** [comment]
It looks there may be some difference between https://github.com/rails/rails/compare/2c3332c...e919a00

The last successful build is https://travis-ci.org/rsim/oracle-enhanced/jobs/545598716
The first failed build is https://travis-ci.org/rsim/oracle-enhanced/builds/545973854

**Comment by yahonda** [comment]
According to `git bisect`, this failure is triggered by 7cc27d749c3563e6b278ad01d233cb92ea3b7935 rails/rails#36439 . We will need to suppor this change.

**Comment by yahonda** [comment]
It also reproduces with release60 branch since https://github.com/rails/rails/commit/e94dc52b2a2ce06480775b7561bd7194da3d2f96

**Comment by yahonda** [comment]
It has been addressed by rails/rails#36525.
